### DISCUSSION TITLE
Git commits error

### DISCUSSION THREAD

**Discussion by alanmvh** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#238] [Q&A]
Buena noche companeros explorers!!!! 🚀🚀🚀🚀

Tengo una grandisima duda a ver si alguien me puede apoyar. Durante la semana en curso estuve trabajando en las practicas de abogabot y html haciendo commits hacia mi repositorio... Pero gran sorpresa!!! no tenia configurado el user.name ni el user.email en git porque formatee mi computador hace unos dias y mi actividad no se vio reflejada en el tablero de mi perfil:
![image](https://user-images.githubusercontent.com/34989670/156697644-7fcd240b-f2e0-49eb-b3ca-a4ff995efaac.png)
 
Si observan en el tablero hay una brecha de inactividad, lo que quisiera es actualizar los commits de ese rango de tiempo con mi user.name y mail pero no logro encontrar como.

**Comment by JuanSalvador73** [comment]
¿Ya le escribiste al Admin de Git?

**Comment by doguedogue** [comment]
Si en tu local hiciste los commits, tienes la historia y la puedes revisar con:
`git log`

Al hacer el push a github se cargará esa información
`git push`

También es posible que puedas añadir cosas a tu último commit
`git commit --amend`

Pero si no los cargaste ni en local,  pues súbelo cuanto antes con tu usuario & email
```
git config --global user.name "XXXX"
git config --global user.email YYYY@ZZZ.com
```

Siempre puedes validar como está tu configuración de git en tu local
`git config --list`
